test/EmptyApp.FunctionalTests/project.json

```diff
@@ -10,7 +10,7 @@
     "Microsoft.AspNetCore.Diagnostics": "1.0.0-rc2-final",
     "FunctionalTestUtils": { "target": "project" },
     "Microsoft.NETCore.Platforms": "1.0.1-*",
-    "dotnet-test-xunit": "1.0.0-*",
+    "dotnet-test-xunit": "1.0.0-rc2-187791-49",
```

Comment:
1.0.0-* will bring the latest bits available. Unless we are sure that the issue is fixed in the latest version, we don't want to use it. Its a testing package, and I was told to use this package that works for sure. So, hard coded the version for this one. Furthermore, I believe this change should only stay in develop branch. Please let me know if that makes sense to have it only in develop. Thanks!